Dockerfile

```diff
@@ -68,6 +69,12 @@ RUN apt-get update -y && \
 WORKDIR /code
 COPY . /code
 
+# Replace the `magick` (imagemagick v7) with a custom script that calls
```

Comment:
There is no `magick` command by default in this Ubuntu 20.04 image, so better say "Add" instead of "Replace" command. Kind of like:

- Add `magick` wrapper command to simulate ImageMagick v7 that is necessary by Snakemake to produce thumbnails in generated reports. The wrapper simply passes conversion requests to ImageMagick v6 available on Ubuntu 20.04.

Reply:
Please also add a news item in CHANGES file saying something like:

- Fixes creation of image thumbnails in Snakemake HTML execution reports.